pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -118,6 +118,14 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
+.PHONY: test-coverage
```

Comment:
The coverage today is done here: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/Makefile#L58-L60

So, if we create a new target only for that we need to update the make test target.
I'd prefer we change the make test target for ignoring the file

Also, I do not think that we can address this need by doing the cat/grep/rm.
We need to look for how we properly ignore files with go test for the option -coverprofile cover.out instead.
What are the available options provided by go test to do it?
Did we check that?

/hold